docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "http instrument pyroscope"
+descriptiontion: "k6 pyroscope instrumentation API"
+weight: 04
+weight: 04
+---
+
+# Pyroscope
```

Comment:
Usually, the page title (`Pyroscope`) matches the title: (`http instrument pyroscope`) field in the front mater.  Also, as a product name, Pyroscope should be capitalized, unless the capitalization is part of the integration name. 

My suggestion here is to adjust the page title and the front matter title: field to match so they make sense in the context of where they are in the docs. 

@heitortsergent do you have any suggestions here?

Reply:
@mstoykov Would this heading also change then? 
```suggestion
# http instrumentation for Pyroscope
```